staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/conversion.go

```diff
@@ -252,7 +252,6 @@ func Convert_map_to_unversioned_LabelSelector(in *map[string]string, out *LabelS
 	if in == nil {
 		return nil
 	}
-	out = new(LabelSelector)
```

Comment:
what issue is this fixing? is there a test for it?

Reply:
This line was causing the whole function to be a no-op, because it reassigns `out` to point to something else, making it no longer an output parameter.

This is now tested as part of `TestReplicationControllerConversion` above. I'll check if there's a unit test for this particular function and add one if not.